pkgs/by-name/la/lammps-mpi/package.nix

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
The definition of `lamps-mpi` should probably still go into `all-packages.nix` (if I read this correctly: https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme)

Reply:
https://github.com/NixOS/nixpkgs/pull/342621

Reply:
@markuskowa I moved the `lammps-mpi` move away from `all-packages.nix` to https://github.com/NixOS/nixpkgs/pull/342621 . As I want to get the important commits of this PR merged, as it seems we agree upon them. I rebased, and removed that commit.